.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "trailingComma": "all",
+  "arrowParens": "avoid",
+  "tabWidth": 2,
+  "bracketSameLine": false,
+  "importOrderCaseInsensitive": true,
+  "semi": true,
+  "singleQuote": false
+}
```

Comment:
I think we might need to add `prettier` (and maybe other associated packages?) in `package.json`.  I'm guess most of us have our editors configured to work with this file already anyway, but good to specify the package requirement still, at least as a dev dependency.